problem-12.c

```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
+// Takes a number as input and determines whether it is an Armstrong number or not
+
+int exponent(int number, int power)
+{
+    int exp = 1;
+    for (int i = 0; i < power; i++)
+    {
+        exp *= number;
+    }
+    return exp;
+}
+
+int main()
+{
+    int number, sum = 0, numberOfDigits = 0;
+
+    printf("---------ARMSTRONG NUMBERS---------\n");
+
+    printf("Enter a number: ");
+    scanf("%d", &number);
+
+    int originalNum = number;
+
+    for (int copyOfNum = number; copyOfNum > 0; copyOfNum /= 10)

```

Comment:
How about just `num` instead of `copyOfNum`? because it is always not a copy of number which is misleading.